eZ/Bundle/EzPublishCoreBundle/Command/CleanupVersionsCommand.php

```diff
@@ -91,15 +100,23 @@ protected function configure()
                 InputOption::VALUE_OPTIONAL,
                 'eZ Platform username (with Role containing at least Content policies: remove, read, versionread)',
                 self::DEFAULT_REPOSITORY_USER
-            );
+            )
+            ->addOption(
+                'excluded-content-types',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Comma separated list of ContentType identifiers of which versions should not be removed, for instance `article`.',
+                self::DEFAULT_EXCLUDED_CONTENT_TYPES
+            )->setHelp(self::BEFORE_RUNNING_HINTS);
```

Comment:
that is not really a help text :)

Reply:
I took this idea from this command: https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Bundle/EzPublishCoreBundle/Command/RegenerateUrlAliasesCommand.php#L76

I have no better idea to display it. How would you like to see it done? 🙂

Reply:
Something like:
```suggestion
                        )->setHelp(
                <<<EOT
The command <info>%command.name%</info> reduces content versions to a minimum. 
It keeps published version untouched, and by default it keeps also the last archived/draft version.
Note: This script can potentially run for a very long time, and in Symfony dev environment it will consume memory exponentially with size of dataset.

{$beforeRunningHints}
EOT
            );
```